### PR TITLE
report page background pic updated

### PR DESCRIPTION
**Pull request by RamakrushnaBiswal**
## Description
report page background pic updated

## Related Issue
#381 solved

## Screenshots / GIFs (if applicable)
image

## Checklist:
- [x] I have performed a self-review of my code
- [x] I have added/updated relevant documentation (if needed)
- [x] I have tested the changes locally and they function as expected
- [x] I have ensured my code follows the project's coding standards

## Additional Notes
[Add any additional notes or context about the changes made]

### CODE REVIEW

**Comment by codewarnab** [comment]
looks good how its looking in mobile ?

**Comment by RamakrushnaBiswal** [comment]
@codewarnab 
image

File: src/Components/shared/Background.css
Comment (codewarnab) [inline src/Components/shared/Background.css:1]: please change this in tailwind and write them in Background jsx  do not create seprate file

File: package-lock.json
Comment (codewarnab) [inline package-lock.json:1]: did you installed a a new npm package ?

**Review by codewarnab — CHANGES_REQUESTED** [review]

File: package-lock.json
Comment (RamakrushnaBiswal) [reply package-lock.json:1]: no

File: src/Components/shared/Background.css
Comment (RamakrushnaBiswal) [reply src/Components/shared/Background.css:1]: ok

Comment (RamakrushnaBiswal) [reply src/Components/shared/Background.css:1]: done [219fea5](https://github.com/Innovateninjas/Paws-frontend/pull/387/commits/219fea56129fa16b878110d942b52b3304e20b2e)

**Review by codewarnab — APPROVED** [review]

**Pull request closed without merging**